README.md

```diff
@@ -149,14 +149,14 @@ Give a ⭐️ if you like this project!
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-Microverse materials and Ruby tutorial at <a href="https://www.tutorialspoint.com/ruby-on-rails/rails-installation.htm">
+The design used for this webapp was created by [Gregoire Vella](https://www.behance.net/gregoirevella) and is based on his [Snapscan](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=) design I would like to thank [Microverse](https://github.com/microverseinc) for the immense support.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] Kindly make a video recording showcasing details of this project, how it works and what it is used for. The recorded video should satisfy the following requirements:

- The video should present the features of the project.
- The maximum length of the video should be 5 minutes.
- The video should contain a clear and objective description of the project in professional English. The topic and purpose of the project should be made clear.
- The video should contain a demo of the project, highlighting all the project features.
- The video should contain a highlight about a piece of the code that you find interesting or you are proud of making.

---
Kindly implement these requirements as they add to your documentation skills. 👍 💻